roles/kubernetes-apps/ansible/defaults/main.yml

```diff
@@ -10,6 +10,7 @@ dns_prevent_single_point_failure: "{{ 'true' if dns_min_replicas|int > 1 else 'f
 enable_coredns_reverse_dns_lookups: true
 coredns_ordinal_suffix: ""
 # dns_extra_tolerations: [{effect: NoSchedule, operator: "Exists"}]
+# dns_extra_nodeselector
```

Comment:
It is necessary to add
```
dns_extra_nodeselector: {}
```
to pass test jobs.

Reply:
@oomichi hello dear
This is [PR](https://github.com/kubernetes-sigs/kubespray/pull/7613) already fixing problem with coredns. Need your approve